components/FeatureHighlight/src/private/MDCFeatureHighlightView+Private.h

```diff
@@ -16,7 +16,7 @@
 
 #import <UIKit/UIKit.h>
 
-#import "MaterialFeatureHighlight.h"
+#import "MDCFeatureHighlightView.h"
```

Comment:
I think this is an incorrect transform.

Reply:
I would disagree for a few reasons:
1. Since this private header extends the API of the `MDCFeatureHighlightView`, it should include its header directly.
1. `MaterialFeatureHighlight` is own component header. Components should not include own headers.

Reply:
I agree with @sigito , but happy to discuss if you feel strongly.